libayatana-appindicator/libayatana-appindicator-gtk3.json

```diff
@@ -0,0 +1,101 @@
+{
+    "name": "libayatana-appindicator",
+    "buildsystem": "cmake-ninja",
+    "config-opts": [
+        "-DENABLE_BINDINGS_MONO=NO",
+        "-DENABLE_BINDINGS_VALA=NO",
+        "-DENABLE_GTKDOC=NO"
+    ],
+    "modules": [
+        "../intltool/intltool-0.51.json",
+        {
+            "name": "libdbusmenu",
+            "buildsystem": "autotools",
+            "build-options": {
+                "cflags": "-Wno-error",
+                "env": {
+                    "HAVE_VALGRIND_FALSE": "#",
+                    "HAVE_VALGRIND_TRUE": ""
+                }
+            },
+            "config-opts": [
+                "--with-gtk=3",
+                "--disable-dumper",
+                "--disable-static",
+                "--disable-tests",
+                "--disable-gtk-doc",
+                "--enable-introspection=no",
+                "--disable-vala"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/libdbusmenu/16.04/16.04.0/+download/libdbusmenu-16.04.0.tar.gz",
+                    "sha256": "b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/libexec",
+                "/lib/pkgconfig",
+                "/lib/*.la",
+                "/share/doc",
+                "/share/libdbusmenu",
+                "/share/gtk-doc",
+                "/share/gir-1.0"
+            ]
+        },
+        {
+            "name": "ayatana-ido",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/AyatanaIndicators/ayatana-ido.git",
+                    "tag": "0.10.4",
+                    "commit": "f968079b09e2310fefc3fc307359025f1c74b3eb",
+                    "x-checker-data": {
+                        "type": "git",
+                        "tag-pattern": "^([\\d.]+)$"
+                    }
+                },
+                {
+                    "type": "shell",
+                    "//": "Remove unused Vala/GIR files",
+                    "commands": [
+                        "sed -i 's/Vala REQUIRED/Vala /' src/CMakeLists.txt",
+                        "sed -i '/install.*AyatanaIdo3-0.4.*/d' src/CMakeLists.txt"
+                    ]
+                }
+            ]
+        },
```

Comment:
this should be a patch

Reply:
and the patch shoud be submitted upstream to remove the `REQUIRED`.

Reply:
Removing the required isn't enough, since find_package still gets run and it fails. Just added a patch to not bother with AyatanaIdo3-0.4.gir if ENABLE_INTROSPECTION is disabled.